src/Shared/ResourceUtilities.cs

```diff
@@ -190,14 +190,27 @@ internal static string FormatResourceString(string resourceName)
         /// <param name="resourceName">Resource string to load.</param>
         /// <param name="args">Optional arguments for formatting the resource string.</param>
         /// <returns>The formatted resource string.</returns>
-        internal static string FormatResourceString(string resourceName, params object[] args)
+        internal static string FormatResourceStringStripCodeAndKeyWord(string resourceName, params object[] args)
         {
             string code;
             string helpKeyword;
 
             return FormatResourceString(out code, out helpKeyword, resourceName, args);
         }
 
+        /// <summary>
+        /// Formats the resource string with the given arguments.
+        /// Ignores error codes and keywords
+        /// </summary>
+        /// <param name="resourceName"></param>
+        /// <param name="args"></param>
+        /// <returns></returns>
+        internal static string FormatResourceStringIgnoreCode(string resourceName, params object[] args)
```

Comment:
Should this be parallel in naming to the above, which says `Strip` instead of `Ignore`?

Reply:
If only there was some git extension / IDE interaction that provided better support for refactorings, as in separating them in extra commits, or at least adding some metadata to the diffs to tie them together and give them a higher level name.